### PR TITLE
feat: Add Purpur tag and configuration file

### PR DESCRIPTION
**Pull request by SkyKingPX**

### CODE REVIEW

File: languages/zh_CN.json [hunk 1]
```diff
@@ -1955,71 +1955,581 @@
   "TXT_CODE_9d1d244f": "程序启动失败，输入输出流不可读：{{pipeName}}",
   "TXT_CODE_ca030197": "实例有其他任务正在进行中，无法进行此操作！",
   "TXT_CODE_aae2918f": "实例进程启动失败，请检查启动命令和运行环境等配置！",
-  "TXT_CODE_2093cc1a": "购买历史查询",
```
Comment (YuMao233) [inline languages/zh_CN.json:1958]: You deleted the existing Chinese translation.

You need to undo the changes to the zh_CN.json file, you only need to change en_US.json.

Comment (SkyKingPX) [reply languages/zh_CN.json:1958]: Do I just add the highlighted one here or do I reset it?

**Comment by SkyKingPX** [comment]
Did you mean like that ^?

File: languages/zh_CN.json (hunk 1 above)
Comment (YuMao233) [reply languages/zh_CN.json:1958]: Reset it

Comment (SkyKingPX) [reply languages/zh_CN.json:1958]: I did. Or the Pufferfish changes too?

**Comment by YuMao233** [comment]
No more questions, thanks for your contribution!

File: languages/zh_CN.json
Comment (SkyKingPX) [inline languages/zh_CN.json:1]: That is not what I did in the Codespace but anyways I fixed it in the next commit

**Comment by YuMao233** [comment]
We will bring your code in the next version~

**Comment by SkyKingPX** [comment]
> We will bring your code in the next version~

Good! Happy to help.

**Comment by YuMao233** [comment]
image
